src/structures/interfaces/InteractionResponses.js

```diff
@@ -224,6 +225,21 @@ class InteractionResponses {
     return options.fetchReply ? this.fetchReply() : undefined;
   }
 
+  /**
+   * Presents a modal component
+   * @param {Modal|ModalOptions} modal The modal to present
+   * @returns {Promise<void>}
+   */
+  async presentModal(modal) {
```

Comment:
Can't we call this `sendModal`? Sounds more natural to me although idk if there's been prior discussion about this or not

Reply:
No specific discussion, just copied the name from the v14 PR. I think I still lean more to "present" which is the terminology commonly used through the web for opening/popping-up a modal.

I know in practice you are of course _sending_ data modal data to the Discord API, but the action is to present it to the user. We don't have sendReply, sendDeferral etc

Reply:
You don’t have sendReply because reply is a verb on its own as well as a noun, however you do say deferReply when deferring. The way I see this is that you *send* the modal to Discord and Discord then presents it to the user. Since we’re not dealing with the web development side of things, presenting isn’t discord.js’s job, as that’s handled by Discord itself. Obviously just a small nit as this is just a name but it’d make more sense to me if it’s called sendModal